.gitignore

```diff
@@ -25,6 +25,8 @@ dist/*
 build/*
 pypaz.egg-info/
 .pytest_cache/
+venv/
+.idea/
```

Comment:
I think this is perfectly fine I am just wondering in which environment are you using these files?

Reply:
venv is the virtual environment folder containing all the executable files related to the project. .idea/ is created in PyCharm to hold some of the project-specific settings.

Reply:
Sounds good.